versioned_docs/version-3.0.0-beta/Edge-Network/Caching.md

```diff
@@ -0,0 +1,42 @@
+---
+id: caching
+title: Caching
+sidebar_label: Caching
+slug: /edge-network/caching
+---
+## Caching 
+
+The Edge Network caches your content at the edge in order to serve data to your users as fast as possible.
+
+Static caching is automatic for all deployments. This means that no changes need to be made to headers. You get the best performance possible out of the box with zero configuration required.
+
+## Cacheable Responses
```

Comment:
Remove this section completely

Reply:
Removed the section, and one quick question are we removing cacheable responses or just that specific heading?